recipes/triqs_cthyb/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs_cthyb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/cthyb/releases/download/{{ version }}/cthyb-{{ version }}.tar.gz
+  sha256: de78b39a354c0e1344987c26403919c23611635ab3d75620aa01bd5b06431bdc
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - triqs 2.2
```

Comment:
Are these versions connected? `2.2.x` of cthyb depends on triqs `2.2`?

Reply:
```suggestion
    - triqs {{ '.'.join(version.split('.')[:2]) }}
```

Reply:
Yes indeed. For these applications we keep the versions locked, at least for the `MAJOR` and `MINOR` part of the version number.

Reply:
Okay, then see my suggestion in my comment above

Reply:
Is this intentionally differen from the run dependency
```
    - {{ pin_compatible("triqs", max_pin="x.x") }}
```